rules/library.bzl

```diff
@@ -68,7 +68,7 @@ module {module_name}.Swift {{
         outputs = [ctx.outputs.destination],
         mnemonic = "ExtendModulemap",
         progress_message = "Extending %s" % ctx.file.source.basename,
-        command = "echo \"$1\" | cat $2 - > $3",
+        command = "echo \"$1\" | cat \"$2\" - > $3",
```

Comment:
Worth quoting $3 as well?

Reply:
Seems responsible. Confirmed it still works as expected.

```
$ bazel build --apple_platform_type=ios //tests/ios/frameworks/mixed-source/custom-module-map:CustomModuleMap
Starting local Bazel server and connecting to it...
INFO: Analyzed target //tests/ios/frameworks/mixed-source/custom-module-map:CustomModuleMap (36 packages loaded, 835 targets configured).
INFO: Found 1 target...
Target //tests/ios/frameworks/mixed-source/custom-module-map:CustomModuleMap up-to-date:
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/CustomModuleMap
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Modules/CustomModuleMap.swiftmodule/x86_64.swiftmodule
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Headers/CustomModuleMap-Swift.h
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Headers/A+CustomModuleMapAdditions.h
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Headers/A.h
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Headers/B+CustomModuleMapAdditions.h
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Headers/B.h
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Headers/CustomModuleMap.h
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/PrivateHeaders/PleaseDontImport.h
  bazel-out/applebin_ios-ios_x86_64-dbg-ST-20a50c27621f/bin/tests/ios/frameworks/mixed-source/custom-module-map/CustomModuleMap/CustomModuleMap.framework/Modules/module.modulemap
INFO: Elapsed time: 73.520s, Critical Path: 35.17s
INFO: 245 processes: 19 internal, 224 darwin-sandbox, 1 local, 1 worker.
INFO: Build completed successfully, 245 total actions
```